offchain/rollups-events/src/main.rs

```diff
@@ -0,0 +1,77 @@
+// Copyright Cartesi Pte. Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
```

Comment:
The strategy that we are using for the test is not trivial, so we should add a little bit of documentation in this file and mention the test file.